R/gmailr.R

```diff
@@ -320,12 +359,11 @@ print.gmail_drafts <- function(x, ...){
 
 the$last_response <- list()
 
-gmailr_query <- function(fun, location, user_id, class = NULL, upload = FALSE,
-                         ...) {
+gmailr_query <- function(fun, location, user_id, class = NULL, ...,
+  upload = FALSE) {
```

Comment:
Was there something fixed by this change?

Reply:
This is a from commit you made in a branch but haven't merged. I found I had to include this commit in order to send mail

6da2a78a819b7763d8987b4b5fb716c759d3ac65

Reply:
Ok, thanks I have merged it now!